src/main/java/com/yahoo/bullet/querying/partitioning/SimpleEqualityPartitioner.java

```diff
@@ -67,7 +67,7 @@
     private static final int LOWEST_BIT_MASK = 1;
     private static final int ZERO = 0;
     // This appends this char to all non-null values to disambiguate them if they actually had NO_FIELD as their values
-    private static final char DISAMBIGUATOR = '.';
+    public static final char DISAMBIGUATOR = '.';
```

Comment:
why changed to public?

Reply:
I'll change it package protected. I used this in a test.